testdata/http_mock/bitstamp/bitstamp.json

```diff
@@ -68270,9 +68270,20 @@
   "/api/withdrawal_requests/": {
    "POST": [
     {
-     "data": [],
+     "data": [{
+        "address": "aMDHooGmAkyrsaQiKhAORhSNTmoRzxqWIO",
+        "amount": "0.00006000",
+        "currency": "BTC",
+        "datetime": "2022-01-31 16:07:32",
+        "id": 1,
+        "network": "bitcoin",
+        "status": 2,
+        "transaction_id": "NsOeFbQhRnpGzNIThWGBTkQwRJqTNOGPVhYavrVyMfkAyMUmIlUpFIwGTzSvpeOP",
+        "txid": 1,
+        "type": 0
+     }],
      "queryString": "",
-     "bodyParams": "key=\u0026nonce=1560480184558170717\u0026signature=5CE4CEFAA663FA03DE0005741683D2AE77D9ACD75F4644A2A1E9030E596D4E04",
+     "bodyParams": "key=&nonce=1701424773861988000&signature=63D3A30E29FF85C814715111F8AC138B0D658C01D45A2EDDCE1E0F09B354DA88&timedelta=1",
```

Comment:
NOTE: nonce is not deterministic so in a mock I cant gain access to this data to test.  Might need a future upgrade to fix this issue.

Reply:
Is nonce not [ignored by the mock URL matcher](https://github.com/thrasher-corp/gocryptotrader/blob/master/exchanges/mock/common.go#L25)?

We could change that to remove ignored fields _before_ doing the len() check, and then mock users can remove these fields from their test when they want to.
Otherwise a nonce of 1.

Reply:
This note above is incorrect, it is all working fine.